manager/src/utils.rs

```diff
@@ -80,6 +80,7 @@ impl Contract {
     }
 
     /// Manage agents
+    // TODO: Check the state changes! getting: Smart contract panicked: The collection is an inconsistent state. Did previous smart contract execution terminate unexpectedly?
```

Comment:
Uh oh, do we remember anything else about this error? Like was it after some used the `manage_agents` function?